src/scss/modifiers/_inverse.scss

```diff
@@ -20,7 +20,7 @@
 			border-color: _oFormsGet('toggle-inverse');
 		}
 
-		input[type=checkbox]:checked { // sass-lint:disable-line no-qualifying-elements
+		input[type=checkbox]:checked { // stylelint-disable selector-no-qualifying-type
```

Comment:
It's not well documented but there is `stylelint-disable-line` and `stylelint-disable-next-line`
https://stylelint.io/changelog#450, which is a little safer (don't disable more than you mean, don't forget to enable again)

Reply:
chatted with Jake, we're coming back to these